admin/services/pupilService.js

```diff
@@ -53,6 +53,16 @@ const pupilService = {
       score,
       percentage
     }
+  },
+
+  sortRecords: async (sortOrder, schoolId) => {
+    let sortedPupils
+    try {
+      sortedPupils = await Pupil.find({ 'school': schoolId }).sort({ 'lastName': sortOrder }).lean().exec()
```

Comment:
I like the efficiency of  `lean`. 👍

Reply:
@jon-shipley will soon post the update not using mongoose as discussed anymore for this